CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 unreleased
 ----------
 - Use version 3.8.0 of braintree-web
+- Fix bug where Card entry would be possible when no cards were supported
```

Comment:
I might change this to something along the lines of "Add support for merchants without cards enabled" because this was a known (albeit not documented) behavior.